corehq/apps/notifications/tests/test_views.py

```diff
@@ -0,0 +1,57 @@
+from unittest.mock import patch
+
+from corehq.apps.accounting.models import Subscription
+from corehq.apps.groups.models import Group
+
+from ..views import NotificationsServiceRMIView
+
+
+def test_should_hide_feature_notifs_for_pro_with_groups():
+    with case_sharing_groups_patch(['agroupid']):
+        hide = NotificationsServiceRMIView._should_hide_feature_notifs("test", "pro")
+        assert hide, "notifications should be hidden for pro domain with groups"
+
+
+def test_should_hide_feature_notifs_for_pro_without_groups():
+    with case_sharing_groups_patch([]), active_service_type_patch("not_IMPLEMENTATION_or_SANDBOX"):
+        hide = NotificationsServiceRMIView._should_hide_feature_notifs("test", "pro")
+        assert not hide, "notifications should not be hidden for pro domain without groups"
+
+
+def test_should_hide_feature_notifs_for_non_pro_with_groups():
+    with case_sharing_groups_patch(['agroupid']), active_service_type_patch("not_IMPLEMENTATION_or_SANDBOX"):
+        hide = NotificationsServiceRMIView._should_hide_feature_notifs("test", None)
+        assert not hide, "notifications should not be hidden for pro domain without groups"
```

Comment:
Non pro domain can't have case sharing groups and should always be shown the feature notification, regardless of if they have a group or not. 

```suggestion
def test_should_hide_feature_notifs_for_non_pro_with_groups():
    with case_sharing_groups_patch([]), active_service_type_patch("not_IMPLEMENTATION_or_SANDBOX"):
        hide = NotificationsServiceRMIView._should_hide_feature_notifs("test", None)
        assert not hide, "notifications should not be hidden for non pro domain with or without groups"
```

Reply:
In that case this test should be removed since it does not make sense. cf35695481b703e49fbc00e286ef6380a8aec394